initramfs/scripts/local-bottom/fixrtc-mount

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh -e
```

Comment:
What will happen if this script returns non-zero for any reason. This feels to clash with the exit 0 and the comment down below.

Reply:
Yeah, I removed the trailing end, this was cargo culted from the original fixrtc script which has the same problem

Reply:
as discussed on IRC ... better remove -e and leave the exit 0 in ... so this script cant block the boot